app/javascript/app/data/initial-state.js

```diff
@@ -1,10 +1,238 @@
 export default {
+  nav: {
+    countriesOpen: false
+  },
+  map: {
+    center: [0, 20],
+    zoom: 1
+  },
   countryExplorer: {
     open: false
   },
   countryNDC: {
     loading: false,
     loaded: false,
     data: {}
+  },
+  countrySelect: {
+    query: '',
+    countries: [
+      // TODO get countries list from carto
```

Comment:
Could we turn this todo comments in issues instead? these comments often end up as garbage in the code... you can keep them along with gh-issues and then remove the comment when you're done if you prefer.